webui/react/src/pages/ExperimentDetails/ExperimentCodeViewer.module.scss

```diff
@@ -0,0 +1,3 @@
+.codeContainer {
+  height: calc(100vh - 220px);
```

Comment:
didnt notice it in the prev review, but when there's no one line log viewer, code viewer is out of window.

<img width="1490" alt="image" src="https://github.com/determined-ai/determined/assets/109113616/76ccbca6-9556-4969-b791-1722517efcdb">

Reply:
Multi-trial experiments have the additional row above tabs with searcher metric, searcher, etc.  So there could be some logic to shrink the code viewer to a different height on multi-trial?  I tried putting the height CSS on the containing Page (in ExperimentDetail) in an earlier version, but that affects scrolling in logs.

Reply:
ok let me know  if the new css class works